contact.rkt

```diff
@@ -75,24 +75,24 @@
 
                 @div[class: "row"]{
                   @div[class: "col-xs-12 col-md-4 pad-contact"]{
-                    @h4{PRL Social} @a[href: "mailto:michelledt@ccs.neu.edu"]{Michelle Thalakottur}, @a[href:"mailto: mhyee@ccs.neu.edu"]{Ming-Ho Yee} @br{}
+                    @h4{PRL Social} @a[href: "mailto:michelledt@ccs.neu.edu"]{Michelle Thalakottur}, @a[href:"mailto:mhyee@ccs.neu.edu"]{Ming-Ho Yee} @br{}
```

Comment:
I'm not doing social anymore.

Reply:
noted, thanks! Wasn't sure since I don't think this job is listed on the wiki.